### DISCUSSION TITLE
mass_genre_updates not reflected in Plex

### DISCUSSION THREAD

**Discussion by rob-bc** [meisnate12/Plex-Meta-Manager#303] [Q&A] [answered]
Including `mass_genre_updates: tmdb` results in a mass update of genres shown in the terminal, but these changes are not reflected in the metadata of the movie in Plex. Am I missing something?

``[2021-06-22 11:51:13,206] [plex_meta_manager.py:323]  [INFO]     | 8 Mile                    | Genres | -Musical, +Music  | ``

![image](https://user-images.githubusercontent.com/48185104/122914189-4a2a5780-d352-11eb-872e-11fb07c65436.png)

**Comment by meisnate12** [comment, marked as answer]
yea it was acting weird but i think ive got it completely working in develop now check it out and let me know

**Reply by rob-bc** [reply]
Getting an error after switching to develop, but it might be to do with my python installation:
```
mms:Plex-Meta-Manager MMS$ python ~/Scripts/Plex-Meta-Manager/plex_meta_manager.py --libraries-only --run
Traceback (most recent call last):
  File "/Users/MMS/Scripts/Plex-Meta-Manager/plex_meta_manager.py", line 6, in <module>
    from modules.builder import CollectionBuilder
  File "/Users/MMS/Scripts/Plex-Meta-Manager/modules/builder.py", line 3, in <module>
    from modules import anidb, anilist, imdb, letterboxd, mal, plex, radarr, sonarr, tautulli, tmdb, trakttv, tvdb, util
  File "/Users/MMS/Scripts/Plex-Meta-Manager/modules/plex.py", line 8, in <module>
    from plexapi.collection import Collection
ImportError: cannot import name 'Collection' from 'plexapi.collection' (/Users/MMS/.pyenv/versions/3.7.0/lib/python3.7/site-packages/plexapi/collection.py)
```

Working fine on master though.

**Reply by meisnate12** [reply]
Install requirements again the plexapi version has increased

**Reply by rob-bc** [reply]
After reinstalling requirements, showing same behaviour as on master

**Reply by rob-bc** [reply]
develop fixed it, thanks!
